dom.html

```diff
@@ -2519,51 +2519,41 @@ <h3 class="heading settled" data-level="4.4" id="interface-node"><span class="se
+           <p>If <var>attr</var> <a data-link-type="dfn" href="#concept-node-equals">equals</a> <var>attr2</var>, then return the result of
+       adding <code class="idl"><a data-link-type="idl" href="#dom-node-document_position_implementation_specific">DOCUMENT_POSITION_IMPLEMENTATION_SPECIFIC</a></code> and <code class="idl"><a data-link-type="idl" href="#dom-node-document_position_following">DOCUMENT_POSITION_FOLLOWING</a></code>. </p>
          </ol>
        </ol>
-      <li>
-       <p>If <var>attr2</var>’s <a data-link-type="dfn" href="#concept-attribute-element">element</a> is non-null, then set <var>node2</var> to <var>attr2</var>’s <a data-link-type="dfn" href="#concept-attribute-element">element</a>. </p>
      </ol>
     <li>
-     <p>If <var>node1</var>’s <a data-link-type="dfn" href="#concept-tree-root">root</a> is not <var>node2</var>’s <a data-link-type="dfn" href="#concept-tree-root">root</a> or <var>node1</var> and <var>node2</var> are <a data-link-type="dfn" href="#concept-attribute">attributes</a>, then return the result of adding <code class="idl"><a data-link-type="idl" href="#dom-node-document_position_disconnected">DOCUMENT_POSITION_DISCONNECTED</a></code>, <code class="idl"><a data-link-type="idl" href="#dom-node-document_position_implementation_specific">DOCUMENT_POSITION_IMPLEMENTATION_SPECIFIC</a></code>, and
+     <p>If <var>node1</var> or <var>node2</var> is null, or <var>node1</var>’s <a data-link-type="dfn" href="#concept-tree-root">root</a> is
+  not <var>node2</var>’s <a data-link-type="dfn" href="#concept-tree-root">root</a>, then return the result of adding <code class="idl"><a data-link-type="idl" href="#dom-node-document_position_disconnected">DOCUMENT_POSITION_DISCONNECTED</a></code>, <code class="idl"><a data-link-type="idl" href="#dom-node-document_position_implementation_specific">DOCUMENT_POSITION_IMPLEMENTATION_SPECIFIC</a></code>, and
   either <code class="idl"><a data-link-type="idl" href="#dom-node-document_position_preceding">DOCUMENT_POSITION_PRECEDING</a></code> or <code class="idl"><a data-link-type="idl" href="#dom-node-document_position_following">DOCUMENT_POSITION_FOLLOWING</a></code>, with the
   constraint that this is to be consistent, together. </p>
      <p class="note no-backref" role="note">Whether to return <code class="idl"><a data-link-type="idl" href="#dom-node-document_position_preceding">DOCUMENT_POSITION_PRECEDING</a></code> or <code class="idl"><a data-link-type="idl" href="#dom-node-document_position_following">DOCUMENT_POSITION_FOLLOWING</a></code> is typically implemented via pointer comparison. In
   JavaScript implementations a cached <code class="lang-javascript highlight"><span class="nb">Math</span><span class="p">.</span><span class="nx">random</span><span class="p">()</span> </code> value can
```

Comment:
Looks like a Bikeshed bug on this line, an extra space before `</code>` that you can see in the rendered result. @tabatkins, known issue?